modules/ding_nodelist/plugins/content_types/ding_nodelist.inc

```diff
@@ -33,6 +33,8 @@ $plugin = array(
 function ding_nodelist_content_type_render($subtype, $conf, $args, $context) {
   $block = new stdClass();
 
+  $block->title = t('Nodelist');
```

Comment:
I recognize what you are doing here using default pane titles as the default values to avoid validation errors. That being said I am not sure that these somewhat technical titles (patrons should not know what a nodelist is) that relate directly to the corresponding module is helpful.

If editors place a pane without changing the title the titles will leak - although only for a short while, 

If we translate the concepts in the module then the translations will carry over to the default node titles and the other way around. I do not think that is helpful.

I wonder whether it would be possible to provide something which is just a bit more friendly to the end-user?

Reply:
I agree with your concern and that we should come up with something more user-friendly. I've posted a comment about it in the issue on redmine.

Reply:
I updated default titles to the suggestions from the issue, which I think are fine seeing as they can always be translated another title, if the library wants to do so.